src/components/templates/PartyMap/components/Map/PartyMapRoomOverlay.tsx

```diff
@@ -14,104 +17,205 @@ import RoomAttendance from "../RoomAttendance";
-            venue.roomVisibility === RoomVisibility.nameCount ||
-            (venue.roomVisibility === RoomVisibility.count &&
-              hasAttendance)) && (
-            <div className="camp-venue-maininfo">
-              {(!venue.roomVisibility ||
-                venue.roomVisibility === RoomVisibility.nameCount) && (
-                <div className="party-map-venue-title">{room.title}</div>
-              )}
-              <RoomAttendance venue={venue} room={room} />
-            </div>
-          )}
+      {/* TODO: this is what Camp's version had: <div className={`camp-venue ${isThisRoomClicked ? "clicked" : ""}`}>*/}
+      {/*<div className={`camp-venue ${isSelectedRoom ? "clicked" : ""}`}>*/}
+      {/* TODO: is this needed here anymore? */}
+      {/*<div className={`grid-room-btn ${isSelectedRoom && "isUnderneath"}`}>*/}
+      {/*  <div*/}
+      {/*      className="btn btn-white btn-small btn-block"*/}
+      {/*      onClick={onJoinRoom}*/}
+      {/*  >*/}
+      {/*    {venue.joinButtonText ?? "Join now"}*/}
+      {/*  </div>*/}
+      {/*</div>*/}
+
+      {/* TODO: rename this class something other than camp-venue-*? */}
+      <div className="camp-venue-img">
+        <img src={room.image_url} title={room.title} alt={room.title} />
+      </div>
+
+      {/* TODO: rename these classes something other than camp-venue-*? */}
+      {shouldShowHovered && hasAttendees && (
+        <div className="camp-venue-text">
+          <div className="camp-venue-maininfo">
+            <div className="party-map-venue-title">{room.title}</div>
+            <RoomAttendance venue={venue} room={room} />
+          </div>
         </div>
+      )}
+
+      <div className={`camp-venue-text`}>
+        {shouldShow2 && (
+          <div className="camp-venue-maininfo">
+            {/* TODO: the definition of shouldShow2 seems to use shouldShow1, so there's probably no point to this check? */}
+            {shouldShow1 && (
+              <div className="party-map-venue-title">{room.title}</div>
+            )}
+            <RoomAttendance venue={venue} room={room} />
+          </div>
+        )}
       </div>
+
+      {/* TODO: this was in Camp MapRoomOverlay, do we need it still?*/}
+      {/*<div className="camp-venue-secondinfo">*/}
+      {/*  <div className="camp-venue-desc">*/}
+      {/*    <p>{room.subtitle}</p>*/}
+      {/*    <p>{room.about}</p>*/}
+      {/*  </div>*/}
+      {/*  <div className="camp-venue-actions">*/}
+      {/*    /!* TODO: extract this into a generalised link component that handles internal/external correctly? *!/*/}
+      {/*    <a*/}
+      {/*      className="btn btn-block btn-small btn-primary"*/}
+      {/*      onClick={onClick3}*/}
+      {/*      href={currentRoomUrl}*/}
+      {/*      {...getExtraLinkProps(isRoomExternal)}*/}
+      {/*    >*/}
+      {/*      {venue.joinButtonText ?? "Enter"}*/}
+      {/*    </a>*/}
+      {/*  </div>*/}
+      {/*</div>*/}
```

Comment:
- [ ] [TODO] this was in `Camp` `MapRoomOverlay`, do we need it still?

Reply:
This seems to be redundant information as we already get it in the modal/similar?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/4b6a1669636861e45d8f7344d9ba82a8437b29c8